ui/pages/nightlies.js

```diff
@@ -182,12 +182,16 @@ const getNightliesByPipeline = async pipeline => {
   return [Object.values(items), latestNightlies];
 };
 const mergeByDate = pipelines => {
-  const today = dayjs();
+  const today = dayjs().utc();
   const runsMerged = new Array(limit);
   for (let i = 0; i < limit; i++) {
     const currentDay = today.add(-i, 'day');
     pipelines.forEach(pipeline => {
-      const matchingRun = pipeline.data.find(run => dayjs(run.startedAt).isSame(currentDay, 'day'));
+      const matchingRun = pipeline.data.find(run => {
+        // Shift by 12 hours to align day boundaries at noon instead of midnight
+        const adjustedDate = dayjs(run.startedAt).utc().subtract(12, 'hours');
```

Comment:
considering the description in the commit this makes sense, but based on this comment it doesn't IMO... consider a pipeline starting at 06:00 in the morning, this will put that pipeline to yesterday - it's not happening right now, but "future person" will check in for 29 additional nightlies running, look at the comment and wonder what's going on...

Reply:
I think you need to do the adjustment in the opposite direction -- the commit message talks about "yesterday night", but the code checks `isSame(currentDay...)`, i.e. "today". So what we need is to move yesterday's afternoon builds into today not today's morning builds into yesterday.